demo-simple/src/main/java/com/novoda/downloadmanager/demo/MainActivity.java

```diff
@@ -111,6 +118,15 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private final VersionOneDatabaseCloner.CloneCallback cloneCallback = updateMessage -> databaseCloningUpdates.setText(updateMessage);
 
+    private final CompoundButton.OnCheckedChangeListener wifiOnlyOnCheckedChange = (buttonView, isChecked) -> {
```

Comment:
Order based on usage when the above is modified too.

Reply:
https://github.com/novoda/download-manager/pull/318/commits/ba90d9f436d912b9b56b09b81c2b52cecaf09052